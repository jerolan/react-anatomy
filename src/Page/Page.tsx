import styled from 'styled-components';

import FlexBox from '../FlexBox';

const Page = styled(FlexBox)`
  min-height: 100vh;
  width: 100vw;
`;

export default Page;
