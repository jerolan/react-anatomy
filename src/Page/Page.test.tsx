import React from 'react';
import renderer from 'react-test-renderer';

import Page from './Page';

describe('<Page />', () => {
  it('renderenders without crashing', () => {
    const component = renderer.create(<Page />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
