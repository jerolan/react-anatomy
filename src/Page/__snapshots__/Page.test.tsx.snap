// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> renderenders without crashing 1`] = `
<div
  className="sc-bdVaJa sc-bwzfXH eGdaqa"
/>
`;
